iree/compiler/Dialect/VM/Conversion/TargetOptions.cpp

```diff
@@ -23,14 +23,19 @@ TargetOptions getTargetOptionsFromFlags() {
       llvm::cl::desc("Bit width of index types."),
       llvm::cl::cat(vmTargetOptionsCategory),
   };
+  static auto *f32ExtensionFlag = new llvm::cl::opt<bool>{
```

Comment:
we should decompose this entirely instead of just pulling out one: `-iree-vm-target-extension-f32` + `-f64` + etc etc
ideally we'd have exclusion expressions in the -iree-vm-target-extension flag (`-f32,+f64`) but that seems like more work

Reply:
Done, and also removed the old flag from lit tests since it's now enabled by default.